app/db/models/index.js

```diff
@@ -44,4 +44,6 @@ Object.keys(db).forEach((modelName) => {
   }
 })
 
+db.sequelize = sequelize
```

Comment:
It's here because we need it later, and this is the only place to export it. There are better / different ways of setting this up, but I took the easiest path.